src/main/resources/assets/ceramics/lang/zh_CN.lang

```diff
@@ -0,0 +1,197 @@
+tile.ceramics.porcelain_barrel.blue.name=蓝色陶瓷桶
+tile.ceramics.porcelain_barrel.purple.name=紫色陶瓷桶
+tile.ceramics.porcelain_barrel.cyan.name=青色陶瓷桶
+tile.ceramics.porcelain_barrel.silver.name=淡灰色陶瓷桶
+tile.ceramics.porcelain_barrel.gray.name=灰色陶瓷桶
+tile.ceramics.porcelain_barrel.pink.name=粉红色陶瓷桶
+tile.ceramics.porcelain_barrel.lime.name=黄绿色陶瓷桶
+tile.ceramics.porcelain_barrel.yellow.name=黄色陶瓷桶
+tile.ceramics.porcelain_barrel.lightBlue.name=淡蓝色陶瓷桶
+tile.ceramics.porcelain_barrel.magenta.name=品红色陶瓷桶
+tile.ceramics.porcelain_barrel.orange.name=橙色陶瓷桶
+tile.ceramics.porcelain_barrel.white.name=白色陶瓷桶
+
+tile.ceramics.porcelain_barrel_extension.black.name=黑色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.red.name=红色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.green.name=绿色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.brown.name=棕色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.blue.name=蓝色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.purple.name=紫色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.cyan.name=青色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.silver.name=淡灰色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.gray.name=灰色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.pink.name=粉红色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.lime.name=黄绿色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.yellow.name=黄色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.lightBlue.name=淡蓝色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.magenta.name=品红色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.orange.name=橙色延展陶瓷桶
+tile.ceramics.porcelain_barrel_extension.white.name=白色延展陶瓷桶
+
+
+# Items
+
+item.ceramics.unfired.bucket.name=未烧制的粘土桶
+item.ceramics.clay_bucket.name=%s粘土桶
+item.ceramics.clay_bucket.empty.name=粘土桶
+item.ceramics.clay_bucket.milk.name=牛奶粘土桶
+item.ceramics.clay_bucket.gravel.name=沙砾粘土桶
+item.ceramics.clay_bucket.sand.name=沙子粘土桶
+item.ceramics.clay_bucket.red_sand.name=红沙粘土桶
+
+item.ceramics.unfired.shears.name=未烧制的粘土剪刀
+item.ceramics.clay_shears.name=粘土剪刀
+
+item.ceramics.unfired.porcelain.name=未烧制的瓷土
+item.ceramics.unfired.porcelain_brick.name=瓷砖
+
+item.ceramics.unfired.clay_plate.name=粘土陶板
+item.ceramics.unfired.clay_plate_raw.name=未烧制的粘土陶板
+
+item.ceramics.clay_helmet_raw.name=未烧制的粘土头盔
+item.ceramics.clay_chestplate_raw.name=未烧制的粘土盔甲
+item.ceramics.clay_leggings_raw.name=未烧制的粘土护腿
+item.ceramics.clay_boots_raw.name=未烧制的粘土靴子
+item.ceramics.clay_helmet.name=粘土头盔
+item.ceramics.clay_chestplate.name=粘土盔甲
+item.ceramics.clay_leggings.name=粘土护腿
+item.ceramics.clay_boots.name=粘土靴子
+
+item.ceramics.unfired.barrel.name=未烧制的陶桶
+item.ceramics.unfired.barrel_extension.name=未烧制的延展陶桶
+item.ceramics.unfired.barrel_porcelain.name=未烧制的陶瓷桶
+item.ceramics.unfired.barrel_porcelain_extension.name=未烧制的延展陶瓷桶
+
+# Messages
+ceramics.barrel.fluid.empty=桶是空的
+ceramics.barrel.fluid.amount=内含：%smb %s
+
+# JEI Descriptions
+ceramics.jei.bucket=粘土桶主要用于还未获得铁桶时的早期游戏，与铁桶一致，它们可以运输流体，但是当倒出炽热的液体时，粘土桶会破碎。
+ceramics.jei.barrel.base=陶桶是一个简单的液体储罐，最多可以存储4桶液体。当被挖掘后，内部存储的液体会丢失。
```

Comment:
Can you use `%s` here and in the line below instead of `4`? (like you did in the porcelain barrels) That way it will display the correct number if its changed in the config